content/altinn-studio/v8/concepts/organisation-library/_index.nb.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Organisasjonsbibliotek
+description: Hva er Altinn 3 bibliotekselementer?
+weight: 10
+---
+
+Altinn Studio Designer organisasjonsbibliotek er et lager for publisering av ressurser som kan brukes av flere Altinn-applikasjoner. De publiserte ressursene
+omtales som bibliotekselementer.
```

Comment:
Ikke alle bibliotekselementer blir nødvendigvis publisert. Vi har for eksempel ingen funksjonalitet for dette i appbiblioteket, og de kan også bli importert.
```suggestion
Altinn Studio Designer organisasjonsbibliotek er et lager for deling av ressurser som kan brukes av flere Altinn-applikasjoner. Ressurene i biblioteket omtales som bibliotekselementer.
```